### DISCUSSION TITLE
P/ Leave taking policy

### DISCUSSION THREAD

**Discussion by 6senseHQ** [6sense-Technologies/6senseHQ-Discussions#10] [General HR]
**For planned leaves:**

Mandatory: Inform the project coordinator before submitting a leave request.
Mandatory: Clear any dependencies with other team members to ensure no tasks are left incomplete due to your absence. Confirm with the project coordinator that all responsibilities have been properly managed.
Mandatory: Notify the project coordinator and the entire team once the leave is approved.

**For emergency or sick leaves:**

Mandatory: Inform the project coordinator & team lead before submitting a leave request.
Mandatory: Notify the project coordinator once the leave is approved.

**Comment by ahsanaasim** [comment]
https://github.com/orgs/6sense-Technologies/discussions/3
